tests/samplesheet_multisample_ont_bam.nf.test.snap

```diff
@@ -402,8 +402,6 @@
                 "HG002_ONT_B_cramino_aligned_phased.arrow:md5,61af72539e105cec79db7c9b78eb15a7",
                 "HG002_ONT_A_cramino_aligned.arrow:md5,d2a5c81595fa34925ab8f03078487d81",
                 "HG002_ONT_B_cramino_aligned.arrow:md5,61af72539e105cec79db7c9b78eb15a7",
-                "HG002_ONT_A_fastqc.html:md5,94d86b38a30f846de64b840656663d18",
-                "HG002_ONT_B_fastqc.html:md5,2ec692ee5acf69717811be481d38f775",
```

Comment:
Why remove this?

Reply:
It contains a date `Mon 28 Oct 2024` so it needs to get a new md5 every day, had me confused for a while 😄

Reply:
Makes sense  :D